doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -52,7 +52,7 @@ From Reference Model section [4.2.4.2.1 Network Acceleration Extensions](../../.
 <a name="2.2.2"></a>
 ### 2.2.2 Network Interface Options
 
-From Reference Model section [4.2.4.4 Network Interface Options](../../../ref_model/chapters/chapter04.md#4244-network-interface-options)
+From Reference Model section [4.2.4.4 Network Interface Options](../../../ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications)
```

Comment:
@prabhubalan The text "[4.2.4.4 Network Interface Options]" also needs to be changed to "[4.2.2 Virtual Network Interface Specifications]"

Reply:
and I think the end of the link shold be "chapter04.md#4.2.2"

Reply:
@pgoyal01 - Thanks. I have updated the model section reference.